generation/WinSDK/emitter.settings.rsp

```diff
@@ -986,7 +986,6 @@ IWICImagingFactory::CreateDecoderFromFilename::pguidVendor=[Optional]
 DCompositionCreateDevice::dcompositionDevice=[ComOutPtr]
 DCompositionCreateDevice2::dcompositionDevice=[ComOutPtr]
 DCompositionCreateDevice3::dcompositionDevice=[ComOutPtr]
-DCompositionGetStatistics::frameStats=[RetVal]
 DCompositionGetTargetStatistics::targetStats=[RetVal]
```

Comment:
```suggestion

```

Reply:
Disagreed. https://github.com/microsoft/win32metadata/pull/1970#discussion_r1736525679.

Reply:
Thanks @riverar. Since we're generating "the same" syntactic sugar for functions as we are for methods, it seems to make sense to have it here.

There are only a handful of `[RetVal]` annotations in `emitter.settings.rsp` currently though.

Reply:
On a separate note, it looks like the header has a broken name:

```cpp
STDAPI DCompositionGetStatistics(
    _In_ COMPOSITION_FRAME_ID frameId,
    _Out_ COMPOSITION_FRAME_STATS* frameStats,
    _In_ UINT targetIdCount,
    _Out_writes_opt_(targetCount) COMPOSITION_TARGET_ID* targetIds,
    _Out_opt_ UINT* actualTargetIdCount);
```

`s/targetCount/targetIdCount` :)

Reply:
And folks keep trying to convince me SAL is used/tested internally. 😜

Reply:
Confusingly `actualTargetIdCount` also has to do with this :)

Reply:
Let's untangle the NativeArrayInfo change here from the revert. This PR should be solely focused on revert.